.github/workflows/_checkmarx.yml

```
@@ -0,0 +1,68 @@
+name: Perform Checkmarx Scan
+
+on:
+  workflow_call: 
+
+    inputs:
+      incremental:
+        description: "Perform incremental scan"
+        type: boolean
+        default: ${{ contains(github.event_name, 'pull_request') }}
+      project_name:
```

```suggestion
      project-name:
```
Fix all of these. Seems the norm is to use dashes.

---

⚠️ I think we shouldn't allow this to be configurable.

---

There is at least one project (i don't remember offhand) which has a special name in checkmarx that does not match it's calculated value. We would need to rename it there, I imagine. But I'm cool with restricting access to the rename. We just need to remember that at least one repo may be weird when we deploy this